moto/autoscaling/models.py

```diff
@@ -496,6 +496,24 @@ def create_or_update_tags(self, tags):
 
             group.tags = new_tags
 
+    def attach_load_balancers(self, group_name, load_balancer_names):
```

Comment:
Once #1255 gets merged, it wouldn't be a bad idea to see if [these](https://github.com/terrycain/moto/blob/19b75770910f2a42158044da9c69ff3783e14a89/moto/awslambda/responses.py#L36-L37) decorators would work with these responses as then you can do things like in [this](https://github.com/terrycain/moto/blob/19b75770910f2a42158044da9c69ff3783e14a89/moto/sqs/responses.py#L287) template which autogenerated a request id (useful for uniqueness when X-ray is being used).

Reply:
Woah, that's pretty slick. I'm definitely willing to roll that in once it's merged. Looks like that should be soon?

Reply:
I've just merged #1255 - let's make it happen :)

Reply:
On it.

Reply:
Done.

Reply:
Done. I also added it to the methods I added in #1264